src/main/kotlin/blackjack/view/BlackJackView.kt

```diff
@@ -0,0 +1,63 @@
+package blackjack.view
+
+import blackjack.domain.BlackJackResult
+import blackjack.domain.Hands
+import blackjack.domain.Player
+
+class BlackJackView {
+    fun askPlayerNames(): List<String> {
+        println("게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)")
+        return readLine()!!.split(",").map { it.trim() }
+    }
+
+    fun startGame(players: List<Player>) {
+        println("${players.map { it.name }.joinToStrings()}에게 2장의 카드를 나누었습니다.")
+        players.forEach {
+            printPlayer(it)
+        }
+        println()
+    }
+
+    fun askPlayerToHit(player: Player): Boolean {
```

Comment:
꼬리 재귀인 경우에는 `tailrec` 키워드를 붙여서 최적화를 시키는게 좋습니다.

```suggestion
    tailrec fun askPlayerToHit(player: Player): Boolean {
```